api/src/main/java/org/pmiops/workbench/api/ConceptsController.java

```diff
@@ -195,12 +192,34 @@ private void addDomainCounts(SearchConceptsRequest request, ConceptListResponse
               (request.getPageNumber() == null) ? 0 : request.getPageNumber());
       if (concepts != null) {
         response.setItems(
-            concepts.getContent().stream().map(TO_CLIENT_CONCEPT).collect(Collectors.toList()));
+            concepts.getContent().stream()
+                .map(ConceptsController::toClientConcept)
+                .collect(Collectors.toList()));
       }
     }
 
     // TODO: consider doing these queries in parallel
     addDomainCounts(request, response);
     return ResponseEntity.ok(response);
   }
+
+  public static Concept toClientConcept(DbConcept concept) {
+    return new Concept()
+        .conceptClassId(concept.getConceptClassId())
+        .conceptCode(concept.getConceptCode())
+        .conceptName(concept.getConceptName())
+        .conceptId(concept.getConceptId())
+        .countValue(concept.getCountValue())
+        .domainId(concept.getDomainId())
+        .prevalence(concept.getPrevalence())
+        .standardConcept(ConceptService.STANDARD_CONCEPT_CODE.equals(concept.getStandardConcept()))
+        .vocabularyId(concept.getVocabularyId())
+        .conceptSynonyms(concept.getSynonyms());
+  }
+
+  private SurveyQuestions toClientSurveyQuestions(DbCriteria dbCriteria) {
+    return new SurveyQuestions()
+        .conceptId(dbCriteria.getLongConceptId())
+        .question(dbCriteria.getName());
```

Comment:
Do surveys have to use this structure or can it be modified to add properties (possibly survey name)? Not needed now, just asking for future reference.

Reply:
Yes, we can definitely add survey name to this model

Reply:
Yes, survey name can definitely be added to the model.